package.json

```diff
@@ -84,12 +84,14 @@
     "**/@types/angular": "^1.6.56",
     "**/@types/hoist-non-react-statics": "^3.3.1",
     "**/@types/chai": "^4.2.11",
-    "**/cypress/@types/lodash": "^4.14.155",
+    "**/cypress/@types/lodash": "^4.14.159",
+    "**/cypress/lodash": "^4.17.20",
```

Comment:
cypress includes lodash as a transitive dependency, locked at "4.17.19". Here I resolve it to "^4.17.20".